shared/components/CommunityAbout/index.scss

```diff
@@ -1,7 +1,14 @@
 // ABOUT REACT LONDON EVENTS
   .CommunityAbout {
-    background-color: $events-color;
     font-size: 16px;
+    @media (min-width: $tablet-portrait-width) {
```

Comment:
Can we order this around to make it a little clearer? So for example move all the @media to the bottom of the CommunityAbout block.

An example could be:

```
  .CommunityAbout {
    font-size: 16px;
    background-color: $events-color;
    padding: 22px 0px;
    &__text {
      margin-bottom: 7px;
      margin-top: 0px;
    }

    &__text--bold {
      margin-top: 0px;
      margin-bottom: 0px;
    }

    @media (min-width: $tablet-portrait-width) {
      font-size: 20px;
    }
    @media (min-width: $tablet-width) {
      font-size: 24px;
    }
    @media (min-width: 768px) {
      padding: 30px 0px;

      &__text {
        margin-bottom: 10px;
        margin-top: 0px;
      }
    }
    @media (min-width: 1024px) {
      padding: 30px 0px 30px 0px
    }
  }

```